src/component/Header.js

```diff
@@ -14,9 +15,9 @@ function Header (props) {
             <nav>
                 <h1>place-my-order.com</h1>
                 <ul>
-                    <li className={pathname === "/" ? "active" : ""}><NavLink to="/">Home</NavLink></li>
-                    <li className={pathname === "/restaurants" ? "active" : ""}><NavLink to="/restaurants">Restaurants</NavLink></li>
-                    <li className={pathname === "/order-history" ? "active" : ""}><NavLink to="/order-history">Order History</NavLink></li>
+                    <li className={pathname === "/" ? "active" : ""}><Link to="/">Home</Link></li>
+                    <li className={pathname.startsWith(`/${constants.pageRestaurants}`) ? "active" : ""}><Link to={`/${constants.pageRestaurants}`}>Restaurants</Link></li>
```

Comment:
this is so much harder to follow than just strings. Also, at least put this into a separate function like `getActiveClass('/restaurants')` or something.